.github/workflows/toolchain_build.yml

```
@@ -0,0 +1,106 @@
+name: Build toolchains
+
+on:
+  push:
+    branches: ['master']
+    tags: ['*']
```

```suggestion
    branches:
      - master
    tags: 
      - "*"
```
I think I see this pattern more frequently but feel free to ignore